src/test/http-mocks/claim-store.ts

```diff
@@ -399,6 +399,18 @@ export function resolveClaimantResponse () {
     .reply(HttpStatus.OK)
 }
 
+export function resolveSettlement () {
```

Comment:
Are these being used anywhere? Don't see usage in this change set.

Reply:
removed